client/MentorPanel/src/components/video.js

```diff
@@ -39,9 +41,10 @@ const VideoPlayer = ({ width, height }) => {
             webkit-playsinline='true'
             config={{
                 file: {
-                    tracks: [
-                        { kind: 'subtitles', src: subtitle_url, srcLang: 'en', default: true },
-                    ]
+                    tracks:
+                        chrome_version === 58
```

Comment:
this will only resolve for a single browser version. Can we find the min working version and disable subtitles for anything less than that version?

Reply:
Is there any easier way to test browser versions than downloading them one by one? Browserstack it seems you have to pay to unlock all browser versions.

Reply:
probably binary search? Check w leeds for what is the lowest known working version number